Lib/test/test_zipapp.py

```diff
@@ -316,6 +317,73 @@ def test_content_of_copied_archive(self):
         with zipfile.ZipFile(new_target, 'r') as z:
             self.assertEqual(set(z.namelist()), {'__main__.py'})
 
+    def test_create_archive_with_include_pattern(self):
```

Comment:
This test isn't needed, as we shouldn't be changing `create_archive`.

Reply:
thank you, i have addressed this